js/scenes/Item.js

```diff
@@ -53,3 +54,16 @@ const styles = StyleSheet.create({
     ...defaultTextStyle,
   },
 });
+
+const mapStateToProps = () => {
+  return {};
+};
+
+const mapDispatchToProps = () => {
+  return {};
+};
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(Item);
```

Comment:
I don't think you need to export a 'raw' and 'reduxed' version of the component for testing. When you're testing I think you can just provide a mock `store` property to the `connect()`ed component and it should 'just work' :)